packages/terra-dialog/package.json

```diff
@@ -32,6 +32,7 @@
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
     "terra-base": "^2.0.0",
+    "terra-clinical-action-header": "^1.6.0",
```

Comment:
Talked with @mjhenkes on this some, and we believe that it might be ideal to move action header to the core repository. terra-core was the building blocks for repos such as terra-clinical and terra-consumer, so it feels like a circular dependency when terra-core relies on stuff set inside one of the repos that extend off of this one.

Reply:
Talked in Scrum. Current decision is to move forward with this review and add themes to the component in Clinical. Later, we are going to evaluate deprecating the clinical ActionHeader and including a new one in core.